Islandora/islandora_solution_pack_newspaper
# Clean up for new code sniffs

**JIRA Ticket**: No JIRA, work related to https://github.com/Islandora/islandora/pull/714

# What does this Pull Request do?

Brings code in line with new code sniffs and cleans up/reuses some testing infrastructure.

# How should this be tested?

Mostly watch for a green on Travis and then check that the tests are being run and really passing.

# Additional Notes:

Example:
* Does this change the interface, add a new feature, or otherwise change behaviours that would require updating documentation? no
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# Interested parties
@jonathangreen  @Islandora/7-x-1-x-committers
